### ISSUE TITLE
[Service Bus Trigger] Make configuration options work for batch functions

### ISSUE THREAD

**Issue by rbedger** [Azure/azure-sdk-for-net#27979]
when using the batch functionality of the ServiceBusTrigger, the function only executes one at a time.  we'd like to be able to pull multiple batches off of a service bus queue and process them concurrently.

disabling the batch mode for the ServiceBusTrigger causes the function to execute in parallel, but we'd like to operate on batches of messages simultaneously instead of individual messages simultaneously.

**Comment by jsquire** [comment]
Thank you for your feedback.  Tagging and routing to the team member best able to assist.

**Comment by JoshLove-msft** [comment]
Lock renewal also doesn't apply to batch functions. Let's use this issue to track the feature request of allowing the same configuration settings that work for single dispatch functions to apply to batch functions.

**Comment by JimSuplizio** [comment]
Hi @rbedger, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
